csharp/device/Microsoft.Azure.Devices.Client/Common/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -134,6 +134,36 @@ static string BuildExpiresOn(TimeSpan timeToLive)
             return Convert.ToString(seconds, CultureInfo.InvariantCulture);
 #endif
         }
+
+#if NETMF
```

Comment:
Hi @markrad,

Could you please consider adding this commented code block to the code?
When I implemented the NETMF port for HTTP I spent sometime figuring the correct way to use the NETMF native crypto classes. As the comment mentions, it's only usable if the assemblies are compiled with those libs included. 
My point is that this should remain in the code for someone building this from the source AND choosing to use the native NETMF crypto libs instead of the managed SHA256 calculation (that takes up flash space and it's less performant).